e2e/v8-coverage/empty-sourcemap/babel.config.js

```diff
@@ -6,5 +6,13 @@
  */
 
 module.exports = {
-  presets: ['@babel/preset-env', '@babel/preset-typescript'],
+  presets: [
+    ['@babel/preset-typescript'],
```

Comment:
@SimenB I'm totally not sure why thing makes the test pass, any pointers?

Reply:
doesn't make much sense - this shouldn't be needed at all

Reply:
it's the same as the others - we either need to not transpile our runner or make sure to transpile it correctly (via adding `targets` since we don't want weird async function transpilation)